dropwizard-logging/src/main/java/smartthings/dw/logging/filter/DynamicThresholdLoggerFilterFactory.java

```diff
@@ -0,0 +1,77 @@
+package smartthings.dw.logging.filter;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.LoggingEvent;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.spi.FilterReply;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.collect.Lists;
+import io.dropwizard.logging.filter.FilterFactory;
+
+import java.util.List;
+
+@JsonTypeName("dynamic")
+public class DynamicThresholdLoggerFilterFactory implements FilterFactory<LoggingEvent> {
+
+    @JsonProperty
```

Comment:
Should these be non-nullable so they catch issues that might be in the config?  Should we default anything here to reasonable values?